### DISCUSSION TITLE
Promotions issues

### DISCUSSION THREAD

**Discussion by studiodrimmel** [swellstores/community#515] [Help]
Hi,

Made some promotions, and they are not being applied to the hosted checkout. 
Created a Buy X get Y -> doesnt work
Order total -> doesnt work either
Category -> does not do anything either

Am I missing something? I expect adding a promotion would update my checkout pages.

Also, the creation of promotions is pretty buggy. Changing a promotion type for example results in console errors
<img width="495" alt="Scherm­afbeelding 2024-03-21 om 11 37 25" src="https://github.com/swellstores/community/assets/101345863/072d1804-67e5-4960-a012-6b6e7a45258c">

**Comment by sags95** [comment]
Are you seeing the discounts array being updated on the cart when inspecting it?

**Reply by TribersNL** [reply]
Nope, not when expending the promotions list. I just created a simple promotion on the order total with a fixed percentage as a test. But no result.

**Reply by sags95** [reply]
@TribersNL Do you have access to our support team on the dashboard? It might be helpful for someone on the team to take a look at your carts/promotions.

**Comment by logeshswell** [comment]
> Also, the creation of promotions is pretty buggy. Changing a promotion type for example results in console errors.

I guess the promotion changes were not saved properly.

**Reply by logeshswell** [reply]
> I expect adding a promotion would update my checkout pages.

Yes, that's the expected behavior and it should reflect in your new cart and older carts (if the cart is updated).

**Reply by logeshswell** [reply]
Can you help me with a screen recording of the dashboard actions that caused the console error?
Also, please share your cart and promotions URL.
